vnext/PropertySheets/React.Cpp.props

```diff
@@ -86,4 +86,6 @@
     </ClCompile>
   </ItemDefinitionGroup>
 
+  <Import Project="$(ReactNativeWindowsDir)PropertySheets\PoisonUnpatchedHeaders.targets" />
```

Comment:
@jonthysell do you know if consuming applications will include this property sheet? I need to add this to somewhere that all projects will include, but am not certain this is the best home.